conan/internal/runner/__init__.py

```diff
@@ -0,0 +1,6 @@
+class RunnerException(Exception):
```

Comment:
Should this exception be defined in `conans/errors.py`? And inherit from `ConanException`?

Reply:
The idea is to be able to see if it is a conan error or only a runner layer error.

Reply:
Got it! Thanks!